packages/ERTP/test/unitTests/test-issuerObj.js

```diff
@@ -44,8 +44,7 @@ test('bad display info', t => {
   const displayInfo = harden({ somethingUnexpected: 3 });
   // @ts-expect-error deliberate invalid arguments for testing
   t.throws(() => makeIssuerKit('fungible', AssetKind.NAT, displayInfo), {
-    message:
-      /key "somethingUnexpected" was not one of the expected keys \["decimalPlaces","assetKind"\]/,
+    message: /Remainder \{"somethingUnexpected":3\} - Must match \{\}/,
```

Comment:
How hard would it be to add more user-supplied error messages to pattern matches?

Reply:
I see some ways to make them marginally better. But I don't know how to make them much better --- to the point that they could compete with bespoke manual checking logic. It is an interesting tradeoff. Good error messages are a huge deal. But so is reliably, and reliably reviewable, input validation. Pattern matching makes much of input validation declarative.

The particular one your commenting on here, regarding unallowed properties, I can improve. Do you want that in this PR? Or (my preference) I file an issue and improve this case and perhaps some others over time?

Reply:
See https://github.com/Agoric/agoric-sdk/pull/5898